iotdb-core/node-commons/src/main/java/org/apache/iotdb/commons/subscription/meta/consumer/ConsumerGroupMeta.java

```diff
@@ -74,6 +74,31 @@ public long getCreationTime() {
     return creationTime;
   }
 
+  public static /* @NonNull */ Set<String> getTopicsUnsubByGroup(
+      final ConsumerGroupMeta currentMeta, final ConsumerGroupMeta updatedMeta) {
+    if (!Objects.equals(currentMeta.consumerGroupId, updatedMeta.consumerGroupId)) {
+      return Collections.emptySet();
+    }
+    if (!Objects.equals(currentMeta.creationTime, updatedMeta.creationTime)) {
+      return Collections.emptySet();
+    }
+    if (!Objects.equals(
+        currentMeta.consumerIdToConsumerMeta, updatedMeta.consumerIdToConsumerMeta)) {
```

Comment:
Consider closing the consumer directly, and the consumerIdToConsumerMeta is altered by the DropConsumerProcedure.... In that case the queue can not be removed.

Reply:
TODO: Remove "topic" key in ConsumerGroupMeta#removeConsumer.